File: 2023/domaci/08-nasobilka/volfmatous/main.cpp
#include <iostream>

void printMultiplicationTable(int n, int count) {
    for (int multiplier = 1; multiplier <= count; multiplier++) {
        std::cout << n * multiplier << std::endl;
    }
}

int main() {
    int n, p;

    std::cin >> n >> p;

    printMultiplicationTable(n, p);

    return 0;
}

File: 2023/domaci/08-nasobilka/volfmatous/main.py
def print_multiplication_table(n: int, count: int) -> None:
    for multiplier in range(1, count + 1):
        print(n * multiplier)


def read_int() -> int:
    return int(input())


if __name__ == "__main__":
    n = read_int()
    p = read_int()

    print_multiplication_table(n, p)
